tests/main.test.js

```diff
@@ -225,6 +225,23 @@ test('accept CSSO options', async () => {
   expect(finalCss).not.toMatch('test css comment');
 });
 
+test('handles extra semicolons', async () => {
+  // Extra semicolons can cause csso.minify() to throw:
+  // [TypeError: Cannot read property '0' of undefined]
+  // https://github.com/peterbe/minimalcss/issues/243
+  // https://github.com/css/csso/issues/378
```

Comment:
I guess 

```js
const { finalCss } = await runMinimalcss('extra-semicolons');
expect(finalCss).toMatch('a{color:red}');
```

would be enough for test, no need check if exception undefined if we check return value